packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -440,6 +423,53 @@ export class AggregatedAttestationPool {
     }
     return attestations;
   }
+
+  private onScrapeMetrics(metrics: Metrics): void {
+    const aggregatedAttestationPoolMetrics = metrics.opPool.aggregatedAttestationPool;
+    const allSlots = Array.from(this.attestationGroupByIndexByDataHexBySlot.keys());
+
+    // always record the previous slot because the current slot may not be finished yet, we may receive more attestations
+    if (allSlots.length > 1) {
+      // same to allSlots[allSlots.length - 2];
```

Comment:
this comment doesn't seem that useful, since we switched the whole code base to `.at`, just need to get used to it, but could note that last item in array is always current slot, hence we want second last item here